src/wp-includes/functions.php

```diff
@@ -2875,64 +2974,73 @@ function wp_die( $message = '', $title = '', $args = array() ) {
  */
 function _default_wp_die_handler( $message, $title = '', $args = array() ) {
 	$defaults = array( 'response' => 500 );
-	$r = wp_parse_args($args, $defaults);
+	$r        = wp_parse_args( $args, $defaults );
 
-	$have_gettext = function_exists('__');
+	$have_gettext = function_exists( '__' );
 
 	if ( function_exists( 'is_wp_error' ) && is_wp_error( $message ) ) {
 		if ( empty( $title ) ) {
 			$error_data = $message->get_error_data();
-			if ( is_array( $error_data ) && isset( $error_data['title'] ) )
+			if ( is_array( $error_data ) && isset( $error_data['title'] ) ) {
 				$title = $error_data['title'];
+			}
 		}
 		$errors = $message->get_error_messages();
 		switch ( count( $errors ) ) {
-		case 0 :
-			$message = '';
-			break;
-		case 1 :
-			$message = "<p>{$errors[0]}</p>";
-			break;
-		default :
-			$message = "<ul>\n\t\t<li>" . join( "</li>\n\t\t<li>", $errors ) . "</li>\n\t</ul>";
-			break;
+			case 0:
+				$message = '';
+				break;
+			case 1:
+				$message = "<p>{$errors[0]}</p>";
+				break;
+			default:
+				$message = "<ul>\n\t\t<li>" . join( "</li>\n\t\t<li>", $errors ) . "</li>\n\t</ul>";
+				break;
 		}
 	} elseif ( is_string( $message ) ) {
 		$message = "<p>$message</p>";
 	}
 
 	if ( isset( $r['back_link'] ) && $r['back_link'] ) {
-		$back_text = $have_gettext? __('&laquo; Back') : '&laquo; Back';
-		$message .= "\n<p><a href='javascript:history.back()'>$back_text</a></p>";
+		$back_text = $have_gettext ? __( '&laquo; Back' ) : '&laquo; Back';
+		$message  .= "\n<p><a href='javascript:history.back()'>$back_text</a></p>";
 	}
 
 	if ( ! did_action( 'admin_head' ) ) :
-		if ( !headers_sent() ) {
+		if ( ! headers_sent() ) {
 			status_header( $r['response'] );
 			nocache_headers();
 			header( 'Content-Type: text/html; charset=utf-8' );
 		}
 
-		if ( empty($title) )
-			$title = $have_gettext ? __('ClassicPress &rsaquo; Error') : 'ClassicPress &rsaquo; Error';
+		if ( empty( $title ) ) {
+			$title = $have_gettext ? __( 'ClassicPress &rsaquo; Error' ) : 'ClassicPress &rsaquo; Error';
+		}
 
 		$text_direction = 'ltr';
-		if ( isset($r['text_direction']) && 'rtl' == $r['text_direction'] )
+		if ( isset( $r['text_direction'] ) && 'rtl' == $r['text_direction'] ) {
 			$text_direction = 'rtl';
-		elseif ( function_exists( 'is_rtl' ) && is_rtl() )
+		} elseif ( function_exists( 'is_rtl' ) && is_rtl() ) {
 			$text_direction = 'rtl';
-?>
+		}
+
+		if ( function_exists( 'language_attributes' ) && function_exists( 'is_rtl' ) ) {
+			$dir_attr = get_language_attributes();
+		} else {
+			$dir_attr = "dir='$text_direction'";
+		}
+		?>
 <!DOCTYPE html>
-<html xmlns="http://www.w3.org/1999/xhtml" <?php if ( function_exists( 'language_attributes' ) && function_exists( 'is_rtl' ) ) language_attributes(); else echo "dir='$text_direction'"; ?>>
+<html xmlns="http://www.w3.org/1999/xhtml" <?php echo $dir_attr; ?>>
```

Comment:
Is the xmlns still needed? (asking for a friend)

Reply:
No, not for HTML5

We can add in a backport of:
https://core.trac.wordpress.org/changeset/48126
Either here or in another PR.

Reply:
Hey, I had props on that one! No wonder it didn't look right.
I don't think it fits as part of this PR.

Reply:
I'll make a noted and get the changeset backported at a later date.